src/codeSigning/SimplifiedSigning.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { generateKeyPair, createSign, createVerify } from 'crypto';
```

Comment:
I think it'd be great if there were a note at the top of this file that briefly explains the signing process and/or provides a link to a document

Reply:
will do.